Makefile

```diff
@@ -5,6 +5,7 @@ include common.mk
 
 DOCKER_IMAGE ?= azul
 DOCKER_TAG ?= latest
+CACHE_SEED ?= ''
```

Comment:
Sure this is the right syntax for creating an empty string, and doesn't create string with two single quotes in it?

Reply:
I saw the fixup but would still like an answer.